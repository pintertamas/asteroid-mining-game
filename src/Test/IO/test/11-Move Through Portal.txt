show
movethroughportal;0
show