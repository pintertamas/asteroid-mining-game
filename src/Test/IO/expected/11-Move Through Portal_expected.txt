movethroughportal;0;successful